Adds example with -AsByteStream for Get-Content. Solves #8896 (#8908)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Import-CsOnlineAudioFile.md

This example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy] (https://docs.microsoft.com/powershell/module/skype/new-csteamscallholdpolicy) to provide the audio file id. 

### -------------------------- Example 4 --------------------------
```powershell
$content = Get-Content "C:\Media\MOH.wav" -AsByteStream -ReadCount 0
$audioFile = Import-CsOnlineAudioFile -ApplicationId 'TenantGlobal' -FileName 'MOH.wav' -Content $content
```

> This example uses the `-AsByteStream` parameter instead of the `-Encoding byte` parameter. Requires PowerShell 6 and later versions, where `-Encoding byte` is no longer available. The example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy](new-csteamscallholdpolicy) to provide the audio file id. 

Reviewer: ```suggestion
This example uses the `-AsByteStream` parameter instead of the `-Encoding byte` parameter. Requires PowerShell 6 and later versions, where `-Encoding byte` is no longer available. The example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy](New-CsTeamsCallHoldPolicy.md) to provide the audio file id. 
```

Sorry @robdy, my mistake.

> This example uses `-AsByteStream` parameter instead `-Encoding byte`. It works in PowerShell 6 and higher version, where `-Encoding byte` is no longer available. The example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy](https://docs.microsoft.com/powershell/module/skype/new-csteamscallholdpolicy) to provide the audio file id. 

Reviewer 2: ```suggestion
This example uses the `-AsByteStream` parameter instead of the `-Encoding byte` parameter. It works in PowerShell 6 and later versions, where `-Encoding byte` is no longer available. The example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy](https://docs.microsoft.com/powershell/module/skype/new-csteamscallholdpolicy) to provide the audio file id. 
```

> This example uses the `-AsByteStream` parameter instead of the `-Encoding byte` parameter. Requires PowerShell 6 and later versions, where `-Encoding byte` is no longer available. The example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy](https://docs.microsoft.com/powershell/module/skype/new-csteamscallholdpolicy) to provide the audio file id. 

Reviewer: ```suggestion
This example uses the `-AsByteStream` parameter instead of the `-Encoding byte` parameter. Requires PowerShell 6 and later versions, where `-Encoding byte` is no longer available. The example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as Music On Hold for Microsoft Teams. The stored variable, $audioFile, will be used with [New-CsTeamsCallHoldPolicy](new-csteamscallholdpolicy) to provide the audio file id. 
```
Better to use short/relative links here.
